app/models/transfer.rb

```diff
@@ -85,6 +84,13 @@ def comission=(value)
     end
   end
 
+  def currency_mismatch?
+    @from_currency = bank_account.try(:currency)
+    @to_currency   = BankAccount.find_by_id(reference_id).try(:currency)
+
+    @from_currency != @to_currency ? true : false
```

Comment:
@Gataniel why can't you just return `@from_currency != @to_currency`? And why do you use class variables here?